samples/bluetooth/broadcast_config_tool/README.rst

```diff
@@ -0,0 +1,676 @@
+.. _broadcast_configuration_tool:
+
+Broadcast Configuration Tool
+############################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Broadcast Configuration Tool application implements the :ref:`BIS gateway mode <nrf53_audio_app_overview>`.
+
+In this mode, transmitting broadcast audio happens using Broadcast Isochronous Stream (BIS) and Broadcast Isochronous Group (BIG).
+The tool uses a shell interface to be able to configure the broadcast source in many different ways.
+
+The following limitations apply to this sample application:
+
+* Audio input: Pre-encoded LC3 data from an SD-card.
+* Maximum two BIG with four BIS streams each.
+
+.. _broadcast_configuration_tool_requirements:
+
+Requirements
+************
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Broadcast Configuration Tool application is designed to be used only with the following hardware:
+
+.. table-from-rows:: /includes/sample_board_rows.txt
+   :header: heading
+   :rows: nrf5340_audio_dk_nrf5340
+
+.. note::
+   The application supports PCA10121 revisions 1.0.0 or above.
+   The application is also compatible with the following pre-launch revisions:
+
+   * Revisions 0.8.0 and above.
+
+.. _broadcast_configuration_tool_ui:
+
+Shell commands list
+**************
+
+The Broadcast Configuration Tool uses a shell interface for all user interactions. The command used is
+
+  .. parsed-literal::
+     :class: highlight
+
+     bct
+
+When describing the commands ``<>`` means that the arguments is mandatory and ``[]`` means that the argument is optional.
```

Comment:
Is this the standard notation?

Reply:
I saw this being used in another documentation in NCS, but I don't know if that is the standard